### PR TITLE
[Dygraph] add docker depoly

### PR DESCRIPTION
**Pull request by WenmuZhou**

### CODE REVIEW

File: deploy/docker/hubserving/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+English | [简体中文](README_cn.md)
+
+## Introduction
+Many user hopes package the PaddleOCR service into an docker image, so that it can be quickly released and used in the docker or k8s environment.
```
Comment (littletomatodonkey) [inline deploy/docker/hubserving/README.md:4]: Many users hope

Comment (littletomatodonkey) [inline deploy/docker/hubserving/README.md:4]: a docker image

File: deploy/docker/hubserving/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This page provide some standardized code to achieve this goal. You can quickly publish the PaddleOCR project into a callable Restful API service through the following steps. (At present, the deployment based on the HubServing mode is implemented first, and author plans to increase the deployment of the PaddleServing mode in the futrue)
```
Comment (littletomatodonkey) [inline deploy/docker/hubserving/README.md:6]: provides

File: deploy/docker/hubserving/README.md (hunk 1 above)
Comment (WenmuZhou) [reply deploy/docker/hubserving/README.md]: done

Comment (WenmuZhou) [reply deploy/docker/hubserving/README.md]: done

File: deploy/docker/hubserving/README.md (hunk 2 above)
Comment (WenmuZhou) [reply deploy/docker/hubserving/README.md]: done

**Pull request merged**